src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1989,6 +2001,51 @@ def make_api_call_with_retries(uri, data, tls_verify, fault_type, summary, cli_e
                 close_subprocess_and_raise_cli_error(clientproxy_process, cli_error + str(e))
 
 
+def fetch_pop_publickey_kid(api_server_port, clientproxy_process):
```

Comment:
better to move this to pop specific class than overloading custom.py

Reply:
Sure, created seperate one named "clientproxyutils" and moved all CSP related utils to that